### PR TITLE
[ie/vtv] Add extractors

### PR DESCRIPTION
**Pull request by DinhHuy2010**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

ADD DESCRIPTION HERE

Add extractor for [VTV Go](https://vtvgo.vn/)
and the [video section](https://vtv.vn/truyen-hinh-truc-tuyen.htm) of vtv.vn
Also support only for vietnamese and videos only (no live broadcast)

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [x] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by seproDev** [comment]
Thank you for the PR, I went ahead and cleaned it up slightly.

File: yt_dlp/extractor/vtv.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+from .common import InfoExtractor
+from ..utils import extract_attributes, get_element_html_by_class, remove_start
+
+
+class VTVGoIE(InfoExtractor):
+    _VALID_URL = [
+        r'(?:https://)?(?:www\.)?vtvgo\.vn/(kho-video|tin-tuc)/[\w.-]*?(?P<id>\d+)(?:\.[a-z]+|/)?(?:$|[?#])',
+        r'(?:https://)?(?:www\.)?vtvgo\.vn/digital/detail\.php\?(?:[^#]+&)?content_id=(?P<id>\d+)',
```
Comment (bashonly) [inline yt_dlp/extractor/vtv.py:8]:
```suggestion
        r'https?://(?:www\.)?vtvgo\.vn/(kho-video|tin-tuc)/[\w.-]*?(?P<id>\d+)(?:\.[a-z]+|/)?(?:$|[?#])',
        r'https?://(?:www\.)?vtvgo\.vn/digital/detail\.php\?(?:[^#]+&)?content_id=(?P<id>\d+)',
```

File: yt_dlp/extractor/vtv.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            'id': video_id,
+            'title': self._og_search_title(webpage, default=None),
+            'description': self._og_search_description(webpage, default=None),
+            'thumbnail': self._og_search_thumbnail(webpage, default=None),
+            'formats': self._extract_m3u8_formats(m3u8_url, video_id, 'mp4'),
+        }
+
+
+class VTVIE(InfoExtractor):
+    _VALID_URL = r'(?:https://)?(?:www\.)?vtv\.vn/video/[\w-]*?(?P<id>\d+)\.htm'
```
Comment (bashonly) [inline yt_dlp/extractor/vtv.py:75]:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?vtv\.vn/video/[\w-]*?(?P<id>\d+)\.htm'
```

**Review by bashonly — APPROVED** [review]

**Comment by DinhHuy2010** [comment]
thank you!

**Pull request merged**